src/services/services.ts

```diff
@@ -1267,6 +1267,7 @@ namespace ts {
         InsertSpaceAfterOpeningAndBeforeClosingNonemptyParenthesis: boolean;
         InsertSpaceAfterOpeningAndBeforeClosingNonemptyBrackets: boolean;
         InsertSpaceAfterOpeningAndBeforeClosingTemplateStringBraces: boolean;
+        InsertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces: boolean;

```

Comment:
Sorry but I'm not sure what you mean by 'optional', do you want me to make it `insert?: boolean` form?